skore/src/skore/sklearn/_plot/metrics/roc_curve.py

```diff
@@ -980,3 +982,64 @@ def _compute_data_for_display(
             ml_task=ml_task,
             report_type=report_type,
         )
+
+    def frame(self, with_auc: bool = False) -> DataFrame:
```

Comment:
For consistency, let's reuse the exact same parameter name than in the constructor.

```suggestion
    def frame(self, with_roc_auc: bool = False) -> DataFrame:
```